test/commands/info.js

```diff
@@ -49,4 +50,23 @@ describe('bower info', function () {
             });
         });
     });
+
+    it('shows help for info when no arg given', function (done) {
+        return Q.Promise(function(resolve, reject) {
+            var info = helpers.command('info');
+
+            helpers.run(info).then(function(loggerEndData) {
+                var commandResult = loggerEndData[0];
+                resolve(commandResult);
+            });
+        })
+        .then(function(commandResult) {
+            // No work should have been done, so no value should be returned.
+            expect(commandResult).to.be(undefined);
+        })
+        .catch(function() {
+            expect().fail('should list info');
+        })
+        .done(done);
+    });
```

Comment:
`helpers.run` returns a [promise](https://github.com/bower/bower/blob/12258324d3962def121764be06f0b3025df0958a/test/helpers.js#L222-L235) so there's no need to import Q and roll your own. You can simplify this test by just doing:

```
it('shows help for info when no arg given', function () {
    return helpers.run(info).then(function(loggerEndData) {
        // No work should have been done, so no value should be returned.
        expect(loggerEndData[0]).to.be(undefined);
    });
});
```

Buuuuuut if you scroll up in the file you'll see that there's already a test doing the [same thing](https://github.com/bower/bower/blob/12258324d3962def121764be06f0b3025df0958a/test/commands/info.js#L32-L36). What you really want to do is check if `cli.createReadOptionsError` is getting called, so try something like:

```
it('throws when no arg given', function () {
    expect(function() {
        info.readOptions([])
    }).to.throwError(/info syntax error/);
});
```